testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/drone/KeycloakDronePostSetup.java

```diff
@@ -102,6 +109,30 @@ private void configureFirefoxDriver(WebDriver driver) {
         }
     }
 
+    private void configureHtmlUnitDriver(WebDriver driver) {
+        if (driver instanceof HtmlUnitDriver) {
```

Comment:
You could use an early return pattern here to reduce nesting, but it's more of a code-style preference, so feel free to ignore.

```java
if (!(driver instanceof HtmlUnitDriver)) {
    return;
}

// The rest of the code
```

Or if you were to use the aforementioned pattern matching:

```java
if (!(driver instanceof HtmlUnitDriver htmlUnitDriver)) {
    return;
}

// The rest of the code, 'htmlUnitDriver' can be accessed here.
```